projenrc/version-bump.ts

```diff
@@ -29,9 +29,17 @@ export class Version {
   public readonly changed: boolean;
 
   constructor() {
-    const tags = exec(['git', 'ls-remote', '--tags', 'origin']);
-    this.current = tags.split('/v').pop()!;
```

Comment:
This was problematic because it doesn't sort by creation date, but rather relies on lexicographical ordering. For example, for `v2.99.0` would be consider "newer" than `v2.220.0`.

Instead, we fetch all tags locally, allowing us to use `--sort` when listing tags. This also ensures the latest tag exists locally because we later run diff against it.